src/auth/index.js

```diff
@@ -118,6 +118,43 @@ export default class Auth {
       .then(responseHandler);
   }
 
+  /**
+   * Exchanges an external token obtained via a native social authentication solution for the user's tokens
+   *
+   * @param {Object} parameters parameters used to obtain user tokens from an external provider's token
+   * @param {String} parameters.subjectToken token returned by the native social authentication solution
+   * @param {String} parameters.subjectTokenType identifier that indicates the native social authentication solution
+   * @param {Object} [parameters.userProfile] additional profile attributes to set or override, only on select native social authentication solutions
+   * @param {String} [parameters.audience] API audience to request
+   * @param {String} [parameters.scope] scopes to request
```

Comment:
As in other `scope` params from [this same file](https://github.com/auth0/react-native-auth0/blob/b6cf3cb8110a349ce96ca14b5512c3e17a42fa57/src/auth/index.js#L166), let's add an example:

```suggestion
   * @param {String} [parameters.scope] scopes requested for the issued tokens. e.g. `openid profile`
```